modules/withRouter.js

```diff
@@ -1,31 +1,43 @@
+import invariant from 'invariant'
 import React from 'react'
 import hoistStatics from 'hoist-non-react-statics'
 import { routerShape } from './PropTypes'
-import warning from './routerWarning'
-
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component'
 }
 
 export default function withRouter(WrappedComponent, options) {
-  const { withRef } = options || {}
```

Comment:
I'd rather do destructuring in case the options grow. If they won't ever (I dunno, maybe?) then this should just be a simple boolean arg.